SwiftProtobuf.xcodeproj/project.pbxproj

```diff
@@ -2090,6 +2096,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				APPLICATION_EXTENSION_API_ONLY = YES;
+				"CODE_SIGN_IDENTITY[sdk=appletvos*]" = "";
```

Comment:
I suspect we don't want to include a code signing identity in our Xcode project files.